### PR TITLE
Fix misspellings of "aren't"

### PR DESCRIPTION
**Pull request by keithc-ca**
Only comment changes.

### CODE REVIEW

File: compiler/z/codegen/OMRMachine.cpp [hunk 1]
```diff
@@ -1207,7 +1207,7 @@ OMR::Z::Machine::assignBestRegisterPair(TR::Register    *regPair,
          }
       }
 
-   // TotalUse & FutureUse are only equal upon the first assignment.  Hence, if they arn't
+   // TotalUse & FutureUse are only equal upon the first assignment.  Hence, if they  aren't
```
Comment (babsingh) [inline compiler/z/codegen/OMRMachine.cpp:1210]: Double space.
```suggestion
   // TotalUse & FutureUse are only equal upon the first assignment. Hence, if they aren't
```

File: compiler/z/codegen/OMRMachine.cpp [hunk 2]
```diff
@@ -1011,7 +1011,7 @@ OMR::Z::Machine::assignBestRegisterSingle(TR::Register    *targetRegister,
    // Have we already assigned a real register
    if (assignedRegister == NULL)
       {
-      // These values are only equal upon the first assignment.  Hence, if they arn't
+      // These values are only equal upon the first assignment.  Hence, if they aren't
```
Comment (babsingh) [inline compiler/z/codegen/OMRMachine.cpp:1014]: Double space.
```suggestion
      // These values are only equal upon the first assignment. Hence, if they aren't
```

File: compiler/z/codegen/OMRMachine.cpp [hunk 3]
```diff
@@ -1189,7 +1189,7 @@ OMR::Z::Machine::assignBestRegisterPair(TR::Register    *regPair,
       freeRegisterHigh = self()->reverseSpillState(currInst, firstReg, toRealRegister(tfreeRegisterHigh));
       }
 
-   // TotalUse & FutureUse are only equal upon the first assignment.  Hence, if they arn't
+   // TotalUse & FutureUse are only equal upon the first assignment.  Hence, if they  aren't
```
[the remark this answers is not in the document]
Comment (keithc-ca) [reply compiler/z/codegen/OMRMachine.cpp:1192]: I thought I fixed those; thanks, @babsingh.

**Comment by keithc-ca** [comment]
I fixed the extra, unwanted spaces I added, but I left the existing (some would say appropriate) double spacing after sentences.

**Review by babsingh — APPROVED** [review]
PR builds are not required since only comments are updated.

**Comment by babsingh** [comment]
Merging since the line endings and copyright checks have passed.

**Pull request merged**